numpy/core/tests/test_conversion_utils.py

```diff
@@ -12,6 +14,12 @@ class StringConverterTestCase:
     case_insensitive = True
     exact_match = False
 
+    def _check_value_error(self, val):
+        with pytest.raises(ValueError) as exc:
+            self.conv(val)
+
+        assert exc.match(r'\(got {}\)'.format(re.escape(repr(val))))
```

Comment:
`with pytest.raises` accepts a [`match` regex](https://docs.pytest.org/en/latest/reference.html#pytest-raises)

Reply:
Guess I expected to need multiple matches or something. Fine to change it, but then the line ends up too long. Your call.

Reply:
I think I agree a bit with matti, you can also put the `exc_regex = r'\(got {}\)'.format(re.escape(repr(val)))` on its own line